setup.py

```diff
@@ -31,7 +31,7 @@ def read_file(filename):
 DEPENDENCY_LINKS = []
 
 setup(name='kinto-dist',
-      version='1.13.0.dev0',
+      version='2.0.0.dev0',
```

Comment:
If we are getting a 2.0 there, I would like to merge the remove-py27 support and upgrade to Python > 3.5 and PostgreSQL > 9.5 at the same time.